src/com/ms/silverking/cloud/dht/client/example/HelloMap.java

```diff
@@ -8,32 +8,29 @@
 import com.ms.silverking.cloud.dht.client.SynchronousNamespacePerspective;
 import com.ms.silverking.cloud.dht.gridconfig.SKGridConfiguration;
 
-public class HelloMap {        
-    
-    public static Map<String, String> runExample(SKGridConfiguration gridConfig) {
-        try {
-            SynchronousNamespacePerspective<String, String>    syncNSP;
-            Map<String,String>    mapA;
-            
-            syncNSP = new DHTClient().openSession(gridConfig)
-                    .openSyncNamespacePerspective("_POTUS", String.class, String.class);
-            
-            mapA = ImmutableMap.of("George Washington", "1789-1797", 
-                                   "John Adams", "1797-1801",
-                                   "Thomas Jefferson", "1801-1809",
-                                   "James Madison", "1809-1817",
-                                   "James Monroe", "1817-1825");
-            syncNSP.put(mapA);
-            return syncNSP.get(mapA.keySet());
-        } catch (Exception e) {
-            throw new RuntimeException(e);
-        }
+public class HelloMap {
+
+  public static Map<String, String> runExample(SKGridConfiguration gridConfig) {
+    try {
+      SynchronousNamespacePerspective<String, String> syncNSP;
+      Map<String, String> mapA;
+
+      syncNSP = new DHTClient().openSession(gridConfig).openSyncNamespacePerspective("_POTUS", String.class,
+          String.class);
+
+      mapA = ImmutableMap.of("George Washington", "1789-1797", "John Adams", "1797-1801", "Thomas Jefferson",
+          "1801-1809", "James Madison", "1809-1817", "James Monroe", "1817-1825");
+      syncNSP.put(mapA);
+      return syncNSP.get(mapA.keySet());
+    } catch (Exception e) {
+      throw new RuntimeException(e);
     }
-    
-    public static void main(String[] args) throws IOException {
-        Map<String,String> mapB = runExample( SKGridConfiguration.parseFile(args[0]) );
-        for (String name : mapB.keySet()) {
-            System.out.printf("%-20s %s\n", name, mapB.get(name));
-        }
+  }
+
+  public static void main(String[] args) throws IOException {
+    Map<String, String> mapB = runExample(SKGridConfiguration.parseFile(args[0]));
+    for (String name : mapB.keySet()) {
+      System.out.printf("%-20s %s\n", name, mapB.get(name));
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**: Iterating over [map.keySet()](https://docs.oracle.com/javase/8/docs/api/java/util/Map.html#keySet--) and calling [map.get(Key)](https://docs.oracle.com/javase/8/docs/api/java/util/Map.html#get-java.lang.Object-) to get the values within the loop is inefficient. 
**Fix**: Iterate over [map.entrySet()](https://docs.oracle.com/javase/8/docs/api/java/util/Map.html#entrySet--) and access the values using [entry.getValue()](https://docs.oracle.com/javase/8/docs/api/java/util/Map.Entry.html#getValue--) to avoid potential performance penalty due to lookup.